xrm/xrm.d.ts

```diff
@@ -344,6 +349,9 @@ declare namespace Xrm
         prependOrgName( sPath: string ): string;
     }
 
+    export module Context {
```

Comment:
I'm assuming @mhegazy that this comment applies to all the extra modules I added, so Xrm.Page.Attribute.RequirementLevel and Xrm.Page.Attribute.SubmitMode should just go in the Xrm.Page module, rather than having an Attribute Module just for them?